src/map/handler/Map.TouchZoom.js

```diff
@@ -109,6 +109,15 @@ L.Map.TouchZoom = L.Handler.extend({
 		    .off(document, 'touchmove', this._onTouchMove)
 		    .off(document, 'touchend', this._onTouchEnd);
 
+		var snap = L.Browser.any3d ? this._map.options.zoomSnap : 1;
+		if (snap) {
```

Comment:
@IvanSanchez I think #4592 will solve the problem with edge. Sorry to disturbed you. Next time I know how to create a correct pull request.